### PR TITLE
Issue #114: waitForCIMessage does not work correctly with message checks.

### PR DESCRIPTION
**Pull request by ggallen**

### CODE REVIEW

File: src/main/java/com/redhat/jenkins/plugins/ci/messaging/ActiveMqMessagingWorker.java [hunk 1]
```diff
@@ -715,46 +723,14 @@ public static String formatMessage (Message message) {
             }
 
             sb.append("Message Content:\n");
```
Comment (scoheb) [inline src/main/java/com/redhat/jenkins/plugins/ci/messaging/ActiveMqMessagingWorker.java]: line above says content...but you are getting message body.

**Comment by scoheb** [comment]
LGTM

**Comment by scoheb** [comment]
@ggallen Can you see if you can address #116 with this PR too?

**Comment by ggallen** [comment]
@scoheb, I will look at it but I want to do it separately.  I have enough trouble with the tests.....

**Comment by ggallen** [comment]
@scoheb , I fixes your issue and one failing test. Can you review?

**Pull request merged**